src/components/dialogs/affiliation-error.vue

```diff
@@ -1,19 +1,19 @@
 <template>
-  <v-dialog v-model="showModal" max-width="60%">
+  <v-dialog v-model="getAffiliationErrorModalVisible" max-width="60%">
     <v-card class="notify-dialog">
-      <v-card-title>
-        <slot name="icon">
-          <v-icon large color="error">mdi-alert</v-icon>
-        </slot>
-        <span>
-          <slot name="title">Error Adding Name Request</slot>
-        </span>
+      <v-card-title class="flex-column">
+        <v-icon large color="error" class="my-4">mdi-alert</v-icon>
+        <span>Error Adding Name Request</span>
       </v-card-title>
-      <v-card-text>
-        <slot name="text">The specified name request has already been affiliated.</slot>
+
+      <v-card-text class="text-center">
+        The specified name request may already be affiliated. Go to
+        <a :href="businessRegistryUrl">your Business Registry</a>
+        to use the name request.
```

Comment:
This text needs to be approved.

Reply:
Janis is OK with this text and link for now. She will update this text depending on the possible error scenarios.